src/gpl/README.md

```diff
@@ -114,19 +114,40 @@ global_placement
 | `-timing_driven_net_weight_max` | Set the multiplier for the most timing-critical nets. The default value is `1.9`, and the allowed values are floats. |
 | `-timing_driven_nets_percentage` | Set the reweighted percentage of nets in timing-driven mode. The default value is 10. Allowed values are floats `[0, 100]`. |
 
+### Debug Mode
+
+The `gpl::global_placement_debug` command initiates a debug mode, enabling real-time visualization of the algorithm's progress on the layout. Use the command prior to executing the `global_placement` command, for example in the `global_place.tcl` script.
```

Comment:
If we are going to make this public then we should promote it out of the gpl:: namespace.

Reply:
Alright, I added it because I could not run the command without the "gpl::"